packages/session/src/index.ts

```diff
@@ -96,18 +127,22 @@ export function createSessionMiddleware(options: SessionOptions) {
         return;
       }
 
-      const needsRotation = sessionCookie?.name && sessionCookie?.name !== primaryCookieName;
+      // Ensure that all known session cookies are set to the same value.
+      const hasAllCookies = writeCookieNames.every((cookieName) => !!cookies[cookieName]);
```

Comment:
Do I understand correctly that we don't strictly need `hasAllCookies`, but this basically means that we don't have to wait as long before we complete the cookie rotation? That is, we could skip this `hasAllCookies` rewriting and instead just wait for a while until we've organically written new versions of all our cookies, but instead we are going to actively write new versions immediately so we don't have to wait so long?

Reply:
That's correct, yes.

Reply:
Excellent, thanks for confirming. This all makes sense to me.